blocks.js

```diff
@@ -2518,6 +2547,14 @@ BlockMorph.prototype.userMenu = function () {
         'userDestroy'
     );
     menu.addItem(
+      'forum code...',
+        function() {
```

Comment:
I vote for "Scratchblocks" over "forum code."  It really is quite Scratch-specific.  When we have our own forum with Snap!blocks we can change the item name.